Neos.Flow/Classes/Http/RequestHandler.php

```diff
@@ -167,6 +167,8 @@ protected function sendResponse()
             header($prepareHeader);
         }
         echo $response->getBody()->getContents();
-        ob_end_flush();
+        while (ob_get_level() > 0) {
```

Comment:
The question is if we rather want to flush and start before the output?

Reply:
Also now looking at it again, this looks more like a quick fix of mine. I think output of a PSR response should probably move to the ResponseInformationHelper ?

Reply:
The logic behind that is unknown to me anyway. Why should anything be buffered? That code was added with PSR-7, AFAICS, but why… I don't know. I'll let you decide.

Reply:
Not sure, feels a bit over the top to have a static method in a Helper to send the PSR response? IMO it should be at the lowest level of a PSR RequestHandler (https://www.php-fig.org/psr/psr-15/#21-psrhttpserverrequesthandlerinterface)

Reply:
My point is that this code as is is not reusable at all, and I guess it would be nice to do that. and @kdambekalns I am pretty sure I jusut took inspiration from other code thtat sents psr messages, eg. other frameworks or psr implementations

Reply:
Hm, still not sure why/where this needs to be reusable. The request handler invokes the middleware/component chain, then outputs the Response it receives to the browser. Why would you do that anywhere else too?

Reply:
own request handler for example. but yeah fine, lets merge this for now :D

Reply:
Thanks, at least logins to Neos are possible again. :)